puppet/spec/classes/slave_spec.rb

```diff
@@ -17,6 +17,13 @@
         else
           it { is_expected.to contain_users__account('jenkins').with_sudo('') }
         end
+
+        if facts[:osfamily] == 'Debian'
+          it { is_expected.to contain_file('/etc/pbuilder/buster64/hooks/F60addforemanrepos') }
+          it { is_expected.not_to contain_file('/etc/pbuilder/buster64/hooks/F65-add-backport-repos') }
```

Comment:
this fails?

Reply:
Correct. I changed the code to ensure absent. Now the tests expect the same.